src/utils/__tests__/yarn.test.js

```diff
@@ -24,7 +28,81 @@ function assertSpawnCalls(expectedProcess, expectedArgs, expectedCwd) {
   expect(spawnCalls[0][2].cwd).toEqual(expectedCwd);
 }
 
+let yarnUserAgent = 'yarn/7.7.7 npm/? node/v8.9.4 darwin x64';
+
 describe('utils/yarn', () => {
+  describe('install()', () => {
```

Comment:
I've written these tests to be more self contained and not do all that funky nested beforeEach business ;P

`jest-expect-contain-deep` ended up being super useful btw, cheers